contrib/opencensus-ext-azure/README.rst

```diff
@@ -114,6 +69,109 @@ You can enrich the logs with trace IDs and span IDs by using the `logging integr
         logger.warning('In the span')
     logger.warning('After the span')
 
+Metrics
+~~~~~~~
+
+The **OpenCensus Azure Monitor Metrics Exporter** allows you to export metrics to `Azure Monitor`_.
```

Comment:
Minor: I feel that we don't need to mention "OpenCensus" here given the context we have in the doc, other wise it is too long?
Up to you, if you decide to keep it, need to update trace/logs to make them consistent.

Reply:
Yes, you are right. We should be consistent with the other two and it seems a bit redundant given the package we are in. I shall remove it.